.github/workflows/pr.yaml

```diff
@@ -93,6 +114,30 @@ jobs:
           platforms: ${{ matrix.platforms }}
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+          
+      - name: Run Trivy in table mode for (${{ matrix.name }})
+        env:
+          TRIVY_NON_SSL: true
+          TRIVY_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-db
+        uses: aquasecurity/trivy-action@0.18.0
+        with:
+          image-ref: registry:5000/fb-output-plugin-${{ matrix.name }}
```

Comment:
what does it refer to? Is it looking at debug image?

Reply:
This action runs for all strategies in the matrix.